install/operator/pkg/generator/assets/install/cluster/syndesis.yml

```diff
@@ -10,10 +10,16 @@ spec:
     kind: Syndesis
     listKind: SyndesisList
     plural: syndesises
-# TODO:  Use "syndeses" when upgrading the CRD version
     singular: syndesis
   scope: Namespaced
-  version: v1alpha1
+  version: v1beta1
+  versions:
+    - name: v1beta1
+      served: true
+      storage: true
+    - name: v1alpha1
+      served: true
+      storage: false
```

Comment:
Have you tried setting `storage` to true in both versions? If that's allowed, you should be able to distinguish CR based on the `APIVersion` field

Reply:
yes it doesn't let me apply the CRD